rplidar.py

```diff
@@ -156,7 +156,7 @@ def stop_motor(self):
         '''Stops sensor motor'''
         self.logger.info('Stoping motor')
         # For A1
-        self._serial_port.setDTR(True)
+        #self._serial_port.setDTR(True)  #here causing conflict with A2. DTR has to be always False to communicate.
         # For A2
         self.set_pwm(0)
```

Comment:
They will be able to call `start_motor()` which will call `setDTR(False)` and `set_pwm(DEFAULT_MOTOR_PWM)`. After that they will be able to change `pwm` if necessary.